documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/websocket/server-websocket.adoc

```diff
@@ -12,6 +12,51 @@
 //
 
 [[pg-server-websocket]]
-=== WebSocket Server Libraries
+=== WebSocket Server
 
-TODO
+// JettyWebSocketServletContainerInitializer.configure(ServletContextHandler context, Configurator configurator); is one entry point.
+
+// JettyWebSocketServlet.configure(JettyWebSocketServletFactory factory); another entry point.
+
+// TODO: why they expose different interfaces? Configurator exposes JettyWebSocketServerContainer, the servlet exposes JettyWebSocketServletFactory => historical, live with it.
```

Comment:
This looks incomplete, TODOs and comments here.

Reply:
Ah yes, they are leftovers. I removed them.